### PR TITLE
fix: `abstract` method

### PR DESCRIPTION
**Pull request by evilebottnawi**
:star:

### CODE REVIEW

File: src/printer.js [hunk 1]
```diff
@@ -526,6 +527,7 @@ function printStatement(path, options, print) {
           ),
           ")",
           node.type ? concat([": ", path.call(print, "type")]) : "",
+          node.isAbstract ? ";" : "",
```
Comment (mgrip) [inline src/printer.js]: should we handle this in `lineShouldEndWithSemiColon` instead?

Comment (evilebottnawi) [reply src/printer.js]: @mgrip it is part of `method` construction and it is not solve in `lineShouldEndWithSemicolon`, we only add semicolon in `lineShouldEndWithSemicolon` after body

Comment (mgrip) [reply src/printer.js]: `lineShouldEndWithSemicolon` gets run on every line of code, and checks if that node type should have a semi colon - in this case i would think if if we see the "method" node type we can just check if its abstract or not, and if it is say that it should end with a semicolon? Don't think its a huge deal, but I think its a nice pattern to have that function be solely responsible for handling semi-colon usage

Comment (evilebottnawi) [reply src/printer.js]: @mgrip let'd do this :+1:

**Comment by evilebottnawi** [comment]
@mgrip done :+1:

**Pull request merged**